client/src/app/site/pages/meetings/modules/user-create-search-selector/components/user-create-search-selector/user-create-search-selector.component.ts

```diff
@@ -0,0 +1,89 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { TranslateService } from '@ngx-translate/core';
+import { BehaviorSubject } from 'rxjs';
+import { RawUser } from 'src/app/gateways/repositories/users';
+import { BaseMeetingComponent } from '../../../../base/base-meeting.component';
+import { ParticipantControllerService } from '../../../../pages/participants/services/common/participant-controller.service';
+import { MeetingComponentServiceCollectorService } from '../../../../services/meeting-component-service-collector.service';
+import { ViewUser } from '../../../../view-models/view-user';
+
+export interface UserSelectionData {
+    userId: number;
+    user?: RawUser;
+}
+@Component({
+    selector: 'os-user-create-search-selector',
+    templateUrl: './user-create-search-selector.component.html',
+    styleUrls: ['./user-create-search-selector.component.scss']
+})
+export class UserCreateSearchSelectorComponent extends BaseMeetingComponent implements OnInit {
+    private _usersSubject = new BehaviorSubject<ViewUser[]>([]);
+
+    @Input()
+    public set usersSubject(input: BehaviorSubject<ViewUser[]>) {
+        this.subscriptions.push(input.subscribe(users => this.usersSubject.next(users)));
+    }
+
+    @Input()
+    public placeholder: string = this.translate.instant(`Select a user...`);
+
+    @Output()
+    public userSelect = new EventEmitter<UserSelectionData>();
```

Comment:
```suggestion
    public userSelected = new EventEmitter<UserSelectionData>();
```

I prefer a past tense verb or a present progressive verb for `EventEmitter`s to signalize that there is something which fires updates.